spec/graphql/language/lexer_spec.rb

```diff
@@ -82,5 +82,25 @@
       rparen_token = tokens[6]
       assert_equal '(RPAREN ")" [1:10])', rparen_token.inspect
     end
+
+    it "counts block string line properly" do
+      str = <<-GRAPHQL
+      """
+      Here is a
+      multiline description
+      """
+      type Query {
```

Comment:
I'm a bit paranoid. I think this works, but could you include another type definition after this one that includes a line break between the `type ..` and the description?

i.e.

```graphql
"""
Test
"""

interface Node {
  id: ID!
}
```

Also maybe include one where the description uses a string without line breaks, i.e.:

```graphql
"Test"

interface Node2 {
  id: ID!
}
```